examples/go.mod

```diff
@@ -6,3 +6,5 @@ require (
 	github.com/cilium/ebpf v0.5.1-0.20210428090942-9aea0e1002a2
 	golang.org/x/sys v0.0.0-20210124154548-22da62e12c0c
 )
+
+replace github.com/cilium/ebpf => ./..
```

Comment:
I don't think this will work: https://github.com/golang/go/issues/30354 The way to go about this is to make the change to examples in a separate PR, after this change has been merged. Otherwise there is no way to update the dependency.

Reply:
It depends on what is `works`. For sure with this patch, I am able to run the kprobe example and am not able to do so without it.